py3status/modules/volume_status.py

```diff
@@ -164,7 +164,8 @@ def toggle_mute(self):
 class PactlBackend(AudioBackend):
     def setup(self, parent):
         if self.device is None:
-            self.device = "0"
+            self.device = check_output(
+                ['pactl', 'list', 'sinks']).decode('utf-8').splitlines()[0][-1]
```

Comment:
Fixed. We use `pactl list short sinks` now. I'm confident it was the same issue because I went out on a hunt to find this issue. Can reproduce this same issue easily by fucking with available devices due to `post_config_hook` only. Temporary fix is ready now. Cheers.
```
lasers~/src/py3status/py3status/modules git:(volume-status-fix-pactl✗) python2 volume_status.py
{'color': '#00FF00', 'full_text': <Composite [{'full_text': u'\u266a: muted'}]>, 'cached_until': 1494498727.0}    
{'color': '#00FF00', 'full_text': <Composite [{'full_text': u'\u266a: muted'}]>, 'cached_until': 1494498731.0}    
{'color': '#00FF00', 'full_text': <Composite [{'full_text': u'\u266a: muted'}]>, 'cached_until': 1494498732.0}    
Traceback (most recent call last):                                                                                
  File "volume_status.py", line 317, in <module>
    module_test(Py3status)
  File "/home/chris/src/py3status/py3status/module_test.py", line 52, in module_test
    print(getattr(module, method)(*method_args))
  File "volume_status.py", line 275, in current_volume
    perc, muted = self.backend.get_volume()
  File "volume_status.py", line 178, in get_volume
    muted, perc = self.re_volume.search(output).groups()                                                          
AttributeError: 'NoneType' object has no attribute 'groups'                                                       
```